### ISSUE TITLE
Selecting multiple objects

### ISSUE THREAD

**Issue by qasimz** [foolwood/SiamMask#89]
What to do if i want to select multiple ROIs?

**Comment by maazullah96** [comment]
I have tried to modify it for multiple ROIs. Maybe I can help you
the issue I am facing is speed, its speed decreases as you select more ROIs

**Comment by qasimz** [comment]
yes, please tell me how i can i modify it or kindly share the code with me?
Do you have gpu?

**Comment by maazullah96** [comment]
I am working on the CPU.
wait for1-2 hours I will contact you back

**Comment by maazullah96** [comment]
https://github.com/maazullah96/files
i have added the files 
sorry can't send you project because i did not  have it now.
if you find any issue I will try my best to help you

**Comment by qasimz** [comment]
I am getting the following error.
OSError: Unable to open file (unable to open file: name = '/home/qasim/SiamMask/experiments/siammask_sharp/resnet50_coco_best_v2.0.1.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)

Where to get that file?

**Comment by maazullah96** [comment]
did your project is working before?

**Comment by qasimz** [comment]
Yes perfectly
________________________________

**Comment by foolwood** [comment]
I will upload a multi-object tracking demo after a workshop challenge.

**Comment by qasimz** [comment]
@foolwood it would be great. Can I ask how much time will it take?

**Comment by qasimz** [comment]
@foolwood any update on mulit-object tracking demo?

**Comment by GeorgeSeif** [comment]
Hi @foolwood, is there any update on the multi-object tracking demo? Super interested in this!

**Comment by SamihaSara** [comment]
> I will upload a multi-object tracking demo after a workshop challenge.

@foolwood   Hi, any update on your workshop? Could you please mention when multi-object tracking demo would be released?

**Comment by marinalpo** [comment]
@foolwood Hello! Could you please upload in the multi-object tracker? I would be really grateful!

**Comment by SamihaSara** [comment]
@foolwood Hello! Will the multi-object tracker version be released in the future?

**Comment by FoxHound0x00** [comment]
@foolwood Any update on the multi-object tracker !?
